ui/src/main/java/com/midtrans/sdk/ui/views/status/PaymentStatusActivity.java

```diff
@@ -107,6 +107,11 @@ private void bindResponse() {
                 case PaymentType.GCI:
                     paymentType.setText(R.string.payment_method_gci);
                     break;
+
+                case PaymentType.TELKOMSEL_CASH:
+                    paymentType.setText(getString(R.string.payment_method_telkomsel_cash));
+                    layoutBank.setVisibility(View.GONE);
```

Comment:
Don't need this line.

Layout bank will be only shown on credit card type payment (see line 144 on updated changes)